test/test_nn.py

```diff
@@ -11016,6 +11016,20 @@ def test_hardswish_grad(self, device):
         inputs.requires_grad = True
         self.assertTrue(gradcheck(F.hardswish, (inputs,)))
 
+    @onlyCPU
```

Comment:
Why is this an only CPU test? Does CUDA do the same thing?

Reply:
https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/cuda/ActivationHardswishKernel.cu#L50 shows that CUDA should not have this issue.

Reply:
Right but it doesn't seem like there's any reason not to test it on CUDA too lol